src/services/usersCRUD/UsersCRUD.go

```diff
@@ -0,0 +1,129 @@
+package usersCRUD
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	"regexp"
+	"log"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/services/common"
+	"github.com/satori/go.uuid"
+)
+
+type Users struct {
```

Comment:
Could you please add json representation? Then you can use :

var newUser Users
json.NewDecoder(r.Body).Decode(&newUser)